src/frontend/index.scss

```diff
@@ -99,3 +99,10 @@ body {
   margin-inline: var(--space-xs) 0;
   color: var(--text-secondary);
 }
+
+#root:has(> .UpdateComponent) {
+  height: 100vh;
+  & > .UpdateComponent {
+    height: 100vh;
+  }
+}
```

Comment:
Couldn't this be simplified into [this](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/5058/files#diff-0243628f3f8faa299456e10249ef062cc6f53901d4327937e04d676bcb28c63dR2) though? It looks like the 100% there is not correct, since switching to 100vh like you have it here works as well